docs/index.rst

```diff
@@ -205,6 +205,70 @@ Topic Management
       .. figure:: images/create_topic.png
          :alt: image
 
+7. When Confluent Monitoring Interceptors are configured on any Kafka-based clients, they write metadata to a topic called ``_confluent-monitoring``. Kafka-based clients include producers, consumers, Connect, KSQL, etc. 
+   |c3| reads that topic to do expert stream monitoring for message delivery assurance and performance for throughput and latency.
```

Comment:
Hard to parse. One too many "and"s or "for"s in here, or perhaps one too few verbs, to understand what is modifying what, grammar-wise. Lest you think I'm being snarky, I'm not -- I do not fully understand this sentence as written.

Reply:
Ack, thanks